manifests/overlays/kubeflow-platform/kubeflow-trainer-roles.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubeflow-trainer-admin
+  labels:
+    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-admin: "true"
+aggregationRule:
+  clusterRoleSelectors:
+    - matchLabels:
+        rbac.authorization.kubeflow.org/aggregate-to-kubeflow-trainer-admin: "true"
+rules: []
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubeflow-trainer-edit
+  labels:
+    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-edit: "true"
+    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-trainer-admin: "true"
+rules:
+  - apiGroups:
+      - kubeflow.org
+    resources:
+      - clustertrainingruntimes
+      - trainingruntimes
+      - trainjobs
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
Actually, I think `kubeflow-trainer-edit` doesn't require to Create, Update, or Delete runtimes, they only need to:
`get`, `list`, `watch`. using the Kubeflow SDK.